### PR TITLE
adding project init

### PR DESCRIPTION
**Pull request by Kayphaz007**
@dolad  please review my pull request

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (dolad) [inline .vscode/settings.json]: remove please

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <li><a href="#backLog">Back Log</a></li>
+        <li><a href="#inProgress">In Progress</a></li>
+        <li><a href="#done">Done</a></li>
+      </ul>
+    </header>
+    <section>
+      <div class="" id="backLog">
+        <h1>Back Log</h1>
+        <div>
+          <img src="./imgs/banana.jpg" alt="" />
```
Comment (dolad) [inline src/index.html]: can you remove the images, the images is not part of the task

Comment (dolad) [reply src/index.html]: Rules number always stick to instruction in engineering

**Review by dolad — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Kayphaz007) [reply .vscode/settings.json]: allright

**Comment by Kayphaz007** [comment]
i have deleted the images folder and vscode settings. @dolad

**Comment by dolad** [comment]
1 more images

**Comment by Kayphaz007** [comment]
i have remove the un necessary images and created card template @dolad

File: src/index.html [hunk 3]
```diff
@@ -8,24 +8,270 @@
...
+                <p>On track</p>
+              </div>
+              <!-- second line -->
+              <div class="card_profile">
+                <p class="card_profile_initials">AA</p>
+                <p class="card_profile_date">27 Feb - 1 Mar</p>
+              </div>
+            </div>
+          </div>
+        </div>
```
Comment (dolad) [inline src/index.html]: you are misusing divs, take a look at tags like sections;
[https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i](url)

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        for (var j=i-1; j >= 0 && arr[j]> currentVal; j--){
+            arr[j+i] = arr[j]
+        }
+        arr[j+i]= currentVal;
+        // console.log(arr);
+    }
+    return arr;
+}
+
+insertionSort([2,1,9,76,4])
```
Comment (dolad) [inline src/index.js]: remove please

**Review by dolad — CHANGES_REQUESTED** [review]

**Comment by Kayphaz007** [comment]
I have fixed the divs misused @dolad

File: src/index.css [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #000235;
+  color: white;
+}
```
Comment (dolad) [inline src/index.css]: this should be in the body, collapse as one

**Review by dolad — APPROVED** [review]

File: src/index.css (hunk 5 above)
Comment (dolad) [reply src/index.css]: @Kayphaz007 please make this changes before you merged

Comment (Kayphaz007) [reply src/index.css]: allright its done

**Pull request merged**